legal-api/src/legal_api/models/alternate_name.py

```diff
@@ -111,6 +119,56 @@ def find_by_name(cls, name: str = None):
         alternate_name = cls.query.filter_by(name=name).filter_by(end_date=None).one_or_none()
         return alternate_name
 
+    @property
+    def office_mailing_address(self):
+        """Return the mailing address."""
+        registered_office = (
+            db.session.query(Office)
+            .filter(Office.legal_entity_id == self.id)
+            .filter(Office.office_type == "registeredOffice")
+            .one_or_none()
+        )
+        if registered_office:
```

Comment:
If AlternateName contains only SP and GP then it won't have any other office_type than businessOffice